scripts/publish-preview.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const { join } = require('path');
+const glob = require('glob').sync;
+const tiny = require('tiny-json-http');
+const fs = require('fs');
+const { gzipSync } = require('zlib');
+
+async function go() {
+	const { TRAVIS_PULL_REQUEST, TRAVIS_PULL_REQUEST_SHA } = process.env;
+
+	if (!TRAVIS_PULL_REQUEST) { throw new ReferenceError('Missing env var TRAVIS_PULL_REQUEST'); }
+	if (!TRAVIS_PULL_REQUEST_SHA) { throw new ReferenceError('Missing env var TRAVIS_PULL_REQUEST_SHA'); }
+
+	const dir = join(__dirname, '..', 'out');
+	const files = glob(join(dir, '**'), { nodir: true });
```

Comment:
nit: `glob` seems like overkill - would `fs.readdirSync('.').filter(x => fs.lstatSync(x).isFile())` or whatever not do the same thing here?

Reply:
We do not know up front how many files reside in subfolders, or how deeply nested they may be. If you’d like to reimplement glob by hand by all means please PR, I don’t have any personal preference for how we do it. 🙏🏻